monkey/infection_monkey/network/relay/utils.py

```diff
@@ -50,36 +42,27 @@ def find_available_island_apis(
 def _find_island_server(
     servers: Iterator[SocketAddress],
     server_results: IslandAPISearchResults,
-    island_api_client_factory: AbstractIslandAPIClientFactory,
 ):
     with suppress(StopIteration):
         server = next(servers)
-        server_results[server] = _check_if_island_server(server, island_api_client_factory)
+        server_results[server] = _check_if_island_server(server)
 
 
-def _check_if_island_server(
-    server: SocketAddress, island_api_client_factory: AbstractIslandAPIClientFactory
-) -> Optional[IIslandAPIClient]:
+def _check_if_island_server(server: SocketAddress) -> bool:
```

Comment:
NIT: `_is_island_server` IMO better hints that it returns a bool and is a shorter name

Reply:
Renamed to `_server_is_island()`. See https://guardicore.atlassian.net/wiki/spaces/RES/pages/2800353435/Infection+Monkey+Development+Guidelines+DRAFT#Functions/Methods-returning-booleans